cryptoid/src/config.ts

```diff
@@ -0,0 +1,6 @@
+import { Requester } from '@chainlink/external-adapter'
+import { Config } from '@chainlink/types'
+
+export const DEFAULT_ENDPOINT = 'difficulty'
+
+export const makeConfig = (prefix?: string): Config => Requester.getDefaultConfig(prefix)
```

Comment:
`makeConfig` needs to return the `apiKey`, something like:
```js
export const makeConfig = (): Config => {
  const apiKey = util.getRandomRequiredEnv('API_KEY')
  return { ...getDefaultConfig(), apiKey }
}
```

Reply:
Isn't it passed from `getDefaultConfig`?

Reply:
Oh right! Sorry